From 33b8423511371d7ef7e68e229f80fd0e817edae9 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Fri, 15 Mar 2019 13:46:48 -0300
Subject: [PATCH] qemuarm64: make u-boot an ATF payload

Disable flash as it breaks ATF and change SYS_TEXT_BASE
to the offset expected by ATF.

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 configs/qemu_arm64_defconfig | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/configs/qemu_arm64_defconfig b/configs/qemu_arm64_defconfig
index 7a4d941b80..a6db85f38c 100644
--- a/configs/qemu_arm64_defconfig
+++ b/configs/qemu_arm64_defconfig
@@ -10,13 +10,14 @@ CONFIG_CMD_BOOTEFI_SELFTEST=y
 CONFIG_CMD_PCI=y
 CONFIG_CMD_USB=y
 CONFIG_OF_BOARD=y
-CONFIG_ENV_IS_IN_FLASH=y
+CONFIG_SYS_TEXT_BASE=0x60000000
+# CONFIG_ENV_IS_IN_FLASH=y
 CONFIG_SCSI_AHCI=y
 CONFIG_AHCI_PCI=y
 CONFIG_BLK=y
 # CONFIG_MMC is not set
-CONFIG_MTD=y
-CONFIG_MTD_NOR_FLASH=y
+# CONFIG_MTD=y
+# CONFIG_MTD_NOR_FLASH=y
 CONFIG_FLASH_CFI_DRIVER=y
 CONFIG_CFI_FLASH=y
 CONFIG_SYS_FLASH_USE_BUFFER_WRITE=y
-- 
2.17.1

